src/Components/Generic/EncounterSheet/PatientBackground/StyledExaminationStatusesWithIcons.js

```diff
@@ -29,6 +32,17 @@ const StyledExaminationStatusesWithIcons = ({
     // handleCreateData(true);
     store.dispatch(setEncounterAndPatient(encounterData, patient));
   };
+  const [docID, setDoc] = React.useState(-1);
+  const createLetter = async () => {
+    let docId = await createSummaryLetter({
+      encounter: encounterData,
+      patientId: patient.id,
+      currentUser,
+      facility,
+    });
+    setDoc(docId);
+    store.dispatch(setEncounterAndPatient(encounterData, patient));
```

Comment:
setEncounterAndPatient function saves the id of the encounter and patient in the store of redux, right? this is not needed and can makes problems (if clicked on prev encounters) in the encounter sheet forms. we need only to open the file in new tab.
![image](https://user-images.githubusercontent.com/17809866/93505837-0b2f2780-f924-11ea-987f-055189deb279.png)
The behavior of setEncounterAndPatient needs to be only in cameraClick function.
Or I missing something...

Reply:
I saw that two but The camera is not disabled and you can open a letter when clicking on it . The problem that occur to me 
as a user is that that a letter can be opened from other encounter but If I am going out for a bit when I come back to the terminal I cannot know which encounter was it . so I added a line to the code that when you are clicking the camera the box of the encounter changes also If you like I can take it out ..... and leave it so you can click everywhere you like and letter will be opened and nobody will know which encounter it was ......

Reply:
I have removed the line   store.dispatch(setEncounterAndPatient(encounterData, patient)); FROM

  const createLetter = async () => {
    let docId = await createSummaryLetter({
      encounter: encounterData,
      patientId: patient.id,
      currentUser,
      facility,
    });
    setDoc(docId);
  };